code/game/objects/items/weapons/storage/wallets.dm

```diff
@@ -57,22 +57,20 @@
 			refreshID()
 
 /obj/item/weapon/storage/wallet/update_icon()
-
+	icon_state = "wallet"
 	if(front_id)

```

Comment:
can't this all just be

if(front_id)
icon_state = "walletid_[front_id.icon_state]" or something